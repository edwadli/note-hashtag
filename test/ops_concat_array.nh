PrintEndline (if { 1 2 } . { 3 4 } == { 1 2 3 4 } then "concat arrays ok" else "concat arrays no")

arr = { 1 2 }
PrintEndline (if arr . arr == { 1 2 1 2 } then "concat arrays to self ok" else "concat ararys to self :(")
  
PrintEndline (if arr . int{} == arr then "concat empty ok" else "concat empty :(")
