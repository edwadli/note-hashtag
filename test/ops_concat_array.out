concat arrays ok
concat arrays to self ok
concat empty ok
